js/extensions/bindings/dynamicDatatableBinding.js

```diff
@@ -0,0 +1,13 @@
+define(['jquery', 'knockout', './datatableBinding'], function($,ko) {
```

Comment:
I'm not really sure I understand the purpose of this binding - could you help me to understand? I see it is used in the pathways-tableview component but not sure why. No objections to it - just trying to understand.

Reply:
Yes. Fun story:  I spent a lot of time trying to re-define the columnDef and columns using the normal 'datatableBinding', but could not get it to work.  Then I found information on the datatables.net forums: https://datatables.net/forums/discussion/34654/add-or-remove-column-dynamically.  This thread calls for calling `destroy()` on the datatable if you are going to make those types of changes.  In the case of the pathway table being dynamic (based on number of steps you want to show), i needed to destroy the table.  I didn't want to hack into our existing datatable binding (it's complicated enough), so instead I 'extended' the current datatableBinding, called it `dynamicDatatableBinding`,  and the entire table is destroyed when the underlying datatable observable (including column defs) changes.

Reply:
Sorry to bring up these painful coding memories during the review! But this is helpful to know and makes sense given the dynamic nature of that data table.

Reply:
No worries, was a fair question, and it is good to document the reasoning behind how it was implemented.  Keep 'em coming.